scanpy/tests/test_read_10x.py

```diff
@@ -18,20 +18,32 @@ def assert_anndata_equal(a1, a2):
 
 
 @pytest.mark.parametrize(
-    ['mtx_path', 'h5_path'],
+    ['mtx_path', 'h5_path', 'prefix'],
     [
         pytest.param(
             ROOT / '1.2.0' / 'filtered_gene_bc_matrices' / 'hg19_chr21',
             ROOT / '1.2.0' / 'filtered_gene_bc_matrices_h5.h5',
+            None,
         ),
         pytest.param(
             ROOT / '3.0.0' / 'filtered_feature_bc_matrix',
             ROOT / '3.0.0' / 'filtered_feature_bc_matrix.h5',
+            None,
+        ),
+        pytest.param(
+            ROOT / '1.2.0' / 'filtered_gene_bc_matrices_prefix' / 'hg19_chr21',
+            ROOT / '1.2.0' / 'filtered_gene_bc_matrices_h5.h5',
+            'prefix_',
+        ),
+        pytest.param(
+            ROOT / '3.0.0' / 'filtered_feature_bc_matrix_prefix',
+            ROOT / '3.0.0' / 'filtered_feature_bc_matrix.h5',
+            'prefix_',
```

Comment:
Instead of adding more example files, could you instead use the files we already have, and copy them with changed names into a temporary directory when the tests are run?

Could you also remove the files you added from the git history?